learningequality/studio
# Use KDropdownMenu to render content nodes menus

## Summary
### Description of the change(s) you made

Replaces VListTile, VMenu, and ContextMenuCloack with KDropdownMenu to show content nodes menus

### Manual verification steps performed
1. Checked the following content node menus:
  * The menus of the items of the main list
  * The menus of the folders tree view
  * The menu that appears when clicking the dropdown icon in the bradcrumbs row
  * The menu that appears on the resource details
2. Checked the following context menus:
  * Right clicked on the items of the main list
  * Right clicked on the items of the folders tree view

### Screenshots (if applicable)

https://github.com/learningequality/studio/assets/51239030/87828250-cec3-4063-983d-d810816e1ad7


___
## Reviewer guidance
### How can a reviewer test these changes?
1. Go and edit a channel
2. Check the menus and context menus listed above.

## References
Closes #4403

## Comments

This should be merged after KDS is released with https://github.com/learningequality/kolibri-design-system/pull/583 merged.

Also, after replacing KDropdownMenu on the content node menus, there will be some issues on keyboard navigation that will be fixed when https://github.com/learningequality/kolibri-design-system/issues/588 is fixed. 

----

### Contributor's Checklist


PR process:


Studio-specifc:



Testing:

___

### Reviewer's Checklist
#### This section is for reviewers to fill out.
